cmd/kubeadm/app/util/apiclient/wait.go

```diff
@@ -100,15 +113,29 @@ func getControlPlaneComponents(cfg *kubeadmapi.ClusterConfiguration) []controlPl
 	if idx == -1 {
 		portScheduler = "10259"
```

Comment:
It's not related to this PR. but it is better to use `contants.XXXPort` instead.

/lgtm

Reply:
/hold

Feel free to cancel it.  It is okay to do in a follow-up PR.

Reply:
yeah, good catch, i should make it use all the constants.